### DISCUSSION TITLE
MWCS processing after REF stack

### DISCUSSION THREAD

**Discussion by yuku0301** [ROBelgium/MSNoise#273] [Q&A]
Thank you for opening the program.
I am currently using MSNoise-1.6.1.
I have a question about MWCS processing after REF stack.

After REF stack by the following command
> msnoise stack -r

In msnoise admin, I have confirmed that 496 MWCS jobs are marked as "todo", but when I run MWCS with the following command, the process does not start, and these jobs are not marked as "done".

> msnoise compute_mwcs

I have confirmed that this problem does not occur with the MOV stack, but with the REF stack. Is it because the STACK jobs are still set to "Inprogress" in the case of REF stack?

I apologize for the inconvenience, but could you please let me know how to solve this problem?

**Comment by asyates** [comment]
Hi,

If I have understood, I think perhaps it is just a misunderstanding of the process.

During the MWCS step, each moving stack is compared with the corresponding reference stack for the station-pair/filter etc., and delay times computed at different lag times. Thus, prior to the MWCS step, it is necessary to compute both the moving stacks and the reference stack (so msnoise stack -r, and msnoise stack -m... with the reset in between). 

So, if you have seen that MWCS is running after you have computed the moving stacks, and outputting results to a 'MWCS' directory, MSNoise is working as expected. If you try to run MWCS without computing the moving stacks (with only the reference), this will not work (unless you set mov=1... at which point it will use the 001DAY stacks from msnoise compute_cc).

Does this help? Or have I misunderstood the problem?
